src/main/java/com/google/sps/model/TasksResponse.java

```diff
@@ -0,0 +1,72 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.model;
+
+import java.util.List;
+
+/**
+ * Contains the summary tasks information that should be passed to the client, as well as the
+ * methods to generate these statistics.
+ */
+public class TasksResponse {
+  private List<String> taskListNames;
```

Comment:
Make these final if possible

Reply:
I assume you meant the class, thanks for pointing that out!

Reply:
Not sure why I didn't do this in the first place, thanks!

Reply:
`RESOLVED`